clumpr/ClusterMain.py

```diff
@@ -145,7 +145,7 @@ def get_median(graph):
 
 def perform_clustering(tree_data, d, t, l, k, args):
     """
-    :param tree_data <tuple>: triple of output to newickFormatReader.getInput()
+    :param tree_data: Output of newickFormatReader.getInput()
```

Comment:
Do you think it would be helpful if you change this comment to `:param tree_data <ReconData>: contains trees and tip mapping information` or do you prefer the current one?